doc/nrf/doc_build.rst

```diff
@@ -16,20 +16,20 @@ Before you can build the documentation, install the |NCS| as described in :ref:`
 Make sure that you have installed the required :ref:`Python dependencies <additional_deps>`.
 
 See :ref:`zephyr:documentation-processors` in the Zephyr developer guide for information about installing the required tools to build the documentation and their supported versions.
-In addition to these tools, you must install `mscgen`_ and make sure the ``mscgen`` executable is in your ``PATH``.
+In addition to these tools, you must install `mscgen`_ and make sure the :file:`mscgen` executable is in your ``PATH``.
```

Comment:
mscgen is not a file name, but the name of the command/executable (and the file name on Windows would be mscgen.exe). So it could either be `` (as code that you enter) or ** (as command name - but we haven't defined that for RST, it's how it displays in DITA).

Reply:
I'll revert to code. I'd prefer to keep ** for PCB and hardware.